adapter/src/hermes/adapter/interceptor.cc

```diff
@@ -0,0 +1,95 @@
+/**
+ * Internal headers
+ */
+#include "interceptor.h"
+
+/**
+ * Define Interceptor list for adapter.
+ */
+#define LIST \
```

Comment:
Could you use a more descriptive name since this token is globally visible?

Reply:
Done.